### PR TITLE
Fix: Allow optional partner params to be sent

### PR DESCRIPTION
**Pull request by ayinloya**
Fixes partners unable to pass in optional params

### CODE REVIEW

File: src/shared.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export interface PartnerParams {
   user_id: string | number;
   job_id: string | number;
   job_type: number;
+  [key: string]: unknown
```
Comment (mldangelo) [inline src/shared.ts:14]: do we only want to support strings? Is there an associated spec change?

**Review by mldangelo — APPROVED** [review]
Don't forget to update the readme!

File: src/shared.ts (hunk 1 above)
Comment (ayinloya) [reply src/shared.ts:14]: Will add an associated spec

Comment (tamssokari) [reply src/shared.ts:14]: waiting on the spec to approve this.

**Review by mldangelo — APPROVED** [review]

**Review by tamssokari — APPROVED** [review]

**Pull request merged**